implementations/index.md

```diff
@@ -315,6 +315,10 @@ includes related resources.
 
 * [json_api_server](https://pub.dartlang.org/packages/json_api_server) is a JSON:API server running on Dart VM.
 
+### <a href="#server-libraries-dart" id="server-libraries-dart" class="headerlink"></a> Crystal
```

Comment:
I think you forgot to update the `href` and `id` attributes.

Reply:
ok, i fixed